src/packages/segmented/doc.taro.md

```
@@ -0,0 +1,74 @@
+| --- | --- | --- | --- |
+| label | 分段项的显示文本 | `ReactNode` | `-` |
+| value | 分段项的值 | `string \| number` | `-` |
+| disabled | 分段项的禁用状态 | `boolean` | `false` |
+| icon | 分段项的显示图标 | `ReactNode` | `-` |
+| className | 自定义类名 | `string` | `-` |
+
+## 主题定制
+
+### 样式变量
+
+组件提供了下列 CSS 变量，可用于自定义样式，使用方法请参考 [ConfigProvider 组件](#/zh-CN/component/configprovider)。
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)